core/cloudflow-operator/src/main/resources/application.conf

```diff
@@ -81,7 +81,7 @@ cloudflow {
       }
 
       spark-runner-driver {
-        requests-memory = "512M"
+        requests-memory = "1024M"
```

Comment:
@maasg  These are only the requests, not the limits. Will the Spark Operator turn this value into a limit somehow?

Reply:
I don't think so. This is a limit and will be treated by the Spark operator as such

Reply:
@agemooij @yuchaoran2011 I think the name used here is for consistency. It does not directly relate to `request/limit` of a k8s pod. 
Here's the configuration flow: 
![IMG_20200407_164849](https://user-images.githubusercontent.com/874997/78683892-2109dc80-78f0-11ea-8e0a-65f3ad54c248.jpg)